src/styl/_components/_preview.styl

```diff
@@ -106,3 +110,158 @@
             left 0
             right 0
             bottom 0
+
+    &-forward // when preview is forward
+        position relative
+
+        .teaser
+            position absolute
+            left 0
+            bottom _rem(30px)
+
+            &-title
+            &-headline
+                color $color-white
+                text-shadow 0 2px 2px $color-base
+
+            &-title
+                position relative
+                z-index 1
+                font-size 1.6em
+                color $color-white
+
+                @media $breakpoints.md
+                    font-size 2.7em
+
+            &-summary
+                display none
+
+        .media
+            .media-wrap
+                &:before
```

Comment:
The article picto (play...) Not visible here,
but I specified it in the description of Colette

Reply:
If it's the picto, where is it inserted?

Reply:
oOo ok i see! But it's really not clear.. I think it's more the skin-video that you should work with, not media-wrap:before

Reply:
![image](https://user-images.githubusercontent.com/19707072/83882457-6efd5e00-a742-11ea-965c-500e38d1d22b.png)

Reply:
Yes I could.
If I wanted to specifically target the "Play" pictogram.

But, as it is a generic modifier in Colette, I considered that when we use the modifier "-forward", all the pictograms contained are centered (video, gallery, podcast, quiz, live ....)

And, in this case, it seems more relevant to me to directly style .media-wrap: before,
rather than all the picto classes existing on the site :
![image](https://user-images.githubusercontent.com/19707072/83883855-5beb8d80-a744-11ea-9b6e-742c39659fcb.png)

Reply:
Hmm.. Ok I see what you mean. So at least a comment? To explain what this is?